File: app/src/main/java/com/bitsanddroids/myplants/plants/Plant.java
package com.bitsanddroids.myplants.plants;
import java.io.Serializable;

import lombok.Data;

@Data
public class Plant implements Serializable {

    private String name;
    private String ImageUrl;
    private int sun;
    private int placement;
    private int toxic;
    private int water;
    private int edible;
    private String plantID;
    private String plantingTime;

    public Plant(){

    }

    public Plant(String name, String imageUrl, int sun, int placement, int toxic, int water, int edible, String plantingTime) {
        this.name = name;
        this.ImageUrl = imageUrl;
        this.sun = sun;
        this.placement = placement;
        this.toxic = toxic;
        this.water = water;
        this.edible = edible;
        this.plantingTime = plantingTime;
    }

}

File: app/src/main/java/com/bitsanddroids/myplants/plants/Plant.kt
package com.bitsanddroids.myplants.plants

import java.io.Serializable


class Plant(val name: String = "", val imageUrl: String = "", val sun: Int = 1
            , val placement: Int = 1, val toxic: Int = 1, val water: Int = 1
            , val edible: Int = 1, val plantingTime: String = "") : Serializable {

    var plantID: String? = null
        set(plantID) {
            field = this.plantID
        }


}

